core/index.py

```diff
@@ -666,3 +666,15 @@ def _solrize_date(d, is_start=True):
     else:
         return None
 
+def get_page_text(page):
+   no_text = ["Text not available"]
+   solr = SolrConnection(settings.SOLR)
+   title, date = page.issue.title, page.issue.date_issued
+   query = 'title:"' + str(title) + '" and date:' + str(date).replace('-', '') + \
```

Comment:
This should use string formatting. Also, it looks like the inputs need to be escaped — e.g. that query looks like it would break if `title` had a `"` character or anything else with meaning to Solr.

Reply:
Added string formatting and escaping for quotes in title